lib/process-services/src/lib/process-list/services/process-filter.service.ts

```diff
@@ -126,6 +127,22 @@ export class ProcessFilterService {
         });
     }
 
+    /**
+     * Creates and returns a filter that matches "running" process instances with index.
+     * @param appId ID of the target app
+     * @returns Filter just created
+     */
+    public getRunningFilterInstanceWithIndex(appId: number): FilterProcessRepresentationModel {
```

Comment:
what if we reuse the getRunningFilterInstance by passing an optional value index: number ?